dpytools/http/upload.py

```diff
@@ -0,0 +1,131 @@
+import datetime
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Tuple, Union
+
+from dpytools.http.base import BaseHttpClient
+
+
+class UploadClient(BaseHttpClient):
+    def __init__(self, upload_url: str):
+        # Inherit backoff_max value from BaseHTTPClient.__init__
+        super().__init__()
+        self.upload_url = upload_url
+        # For security purposes, `self.access_token` should be set using a FLORENCE_TOKEN environment variable
+        self.access_token = os.environ.get("FLORENCE_TOKEN")
+        self.headers = {"X-Florence-Token": self.access_token}
+
+    # TODO The `upload` method is currently configured to match the old `upload` endpoint specification. Once the `upload-new` endpoint has been released, the `params` argument passed to `self.post` will need to be adjusted to the new specification (see https://github.com/ONSdigital/dp-upload-service/blob/b91b53a13b28c6ba1095abee71680c591862c68c/swagger.yaml).
```

Comment:
I think upload-new is turned on for the AND stuff we need to be working with, think you'll need to do it 😢 

I would probably make it a different method, may as well have our client support both given you've written the code, you should be able to pull quite a few things out as part of that.